contracts/tests/psp721.rs

```diff
@@ -223,16 +246,10 @@ mod psp721 {
         assert_eq!(nft.balance_of(accounts.bob), 0);
         // Eve owns 2 tokens.
         assert_eq!(nft.balance_of(accounts.eve), 2);
-
-        change_caller(accounts.alice);
-        // Remove operator approval for Bob on behalf of Alice.
-        assert!(nft.set_approval_for_all(accounts.bob, false).is_ok());
-        // Bob is not an approved operator for Alice.
-        assert_eq!(nft.is_approved_for_all(accounts.alice, accounts.bob), false);
     }
 
     #[ink::test]
-    fn not_approved_transfer_should_fail() {
+    fn not_approved_transfer_works() {
```

Comment:
You need to remove checks for allowance ONLY for `mint` and `burn`, not for `transfer`. `transfer` should work like before=)